spark/src/main/kotlin/com/adevinta/spark/components/bottomsheet/layout/ModalBottomSheetLayout.kt

```diff
@@ -19,7 +19,7 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-package com.adevinta.spark.components.bottomsheet
+package com.adevinta.spark.components.bottomsheet.layout
```

Comment:
Note that changing the package is a breaking change since all imports will fail on our consumers